app/graphql/resolvers/add_on_resolver.rb

```diff
@@ -16,7 +16,7 @@ def resolve(id: nil)
 
       current_organization.add_ons.find(id)
     rescue ActiveRecord::RecordNotFound
-      not_found_error
+      not_found_error(code: 'add_on_not_found')
```

Comment:
What about something like `not_found_error(resource: 'add_on')` instead to not repeat `not_found` each time?